scripts/build-storybook.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+GREEN='\033[0;32m'
+NC='\033[0m' # No color
+
+set -e
+
+OUT="$1"
+
+if [ "$OUT" = "docs" ]; then
+  build-storybook -o packages/docs/static/storybook && \
+  echo "${GREEN}info${NC}: Moving static assets for use with gatsby."
+  mkdir -p packages/docs/static/fonts && \
+  mkdir -p packages/docs/static/images && \
+  cp -R packages/docs/static/storybook/fonts/* packages/docs/static/fonts/ && \
+  cp -R packages/docs/static/storybook/images/* packages/docs/static/images/ && \
```

Comment:
Should we consider ignoring these folders in git so we don't get so many duplicate files in version control?

Reply:
We ignore the `static/storybook` folder already

Reply:
I suppose, will just need to make sure to not ignore them if any updates are made there.